### PR TITLE
Merge main into fluent2-colors

### PR DESCRIPTION
**Pull request by laminesm**
### Platforms Impacted
- [x] iOS
- [x] macOS

### Description of changes

This PR merges `main` into `fluent2-colors`.

### Verification

|Components |Main branch | Fluent2-tokens before                                       | Fluent2-tokens After                                      |
|----|----|----------------------------------------------|--------------------------------------------|
|Color| main_color | before_color | after_color |
|DateTimePicker| main_date_time_picker | before_date_time_picker | after_date_time_picker |
|Drawer| main_drawer | before_drawer | after_drawer |
|ActionsCell| main_other_cells | before_other_cells | after_other_cells |
|PopupMenu| main_popup_menu | before_popup_menu | after_popup_menu |
|TabBar| main_tab_bar | before_tab_bar | after_tab_bar |
|SideTabBar| main_side_tab_bar | before_side_tab_bar | after_side_tab_bar |
|TableViewCell| main_tvc | before_tvc | after_tvc |

### Pull request checklist

This PR has considered:
- [x] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1359)

### CODE REVIEW

File: ios/FluentUI/Divider/DividerTokenSet.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                return .dynamicColor { theme.aliasTokens.strokeColors[.neutral2] }
+            }
+        }
+    }
+
+    /// MSFDividerSpacing enumeration value that will define pre-defined value for the padding.
+    let spacing: () -> MSFDividerSpacing
+
+    /// The default thickness for the divider: half pt.
+    static var thickness: CGFloat { return 0.5 }
```
Comment (harrieshin) [inline ios/FluentUI/Divider/DividerTokenSet.swift:49]: we still don't have the right strokewidth token for this?

File: ios/FluentUI/Divider/DividerTokenSet.swift [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                    switch spacing() {
+                    case .none:
+                        return GlobalTokens.spacing(.none)
+                    case .medium:
+                        return GlobalTokens.spacing(.medium)
+                    }
+                }
+
+            case .color:
+                return .dynamicColor { theme.aliasTokens.strokeColors[.neutral2] }
```
Comment (harrieshin) [inline ios/FluentUI/Divider/DividerTokenSet.swift:40]: just confirming. these are the right stroke colors?

Comment (laminesm) [reply ios/FluentUI/Divider/DividerTokenSet.swift:40]: Technically yes, but will need to be pulled from the `aliasTokens.colors` instead.

File: ios/FluentUI/Divider/DividerTokenSet.swift (hunk 1 above)
Comment (laminesm) [reply ios/FluentUI/Divider/DividerTokenSet.swift]: We do! I've just updated the token.

**Review by harrieshin — APPROVED** [review]

**Pull request merged**